src/components/Table/TableHeader/TableHeader.module.scss

```diff
@@ -1,12 +1,12 @@
 .tableHeader {
   display: grid;
   grid-template-columns: var(--table-grid-template-columns);
+  position: sticky;
+  top: 0;
+  z-index: 1;
```

Comment:
z-index always makes me shake a little bit. I'm just making sure this is necessary.

Reply:
Since we're sticking the header to top, and this z-index is stacked inside the table header, I don't think we have a lot to be worried here (+ I've used the minimum)